lib/MusicBrainz/Server/Controller/WS/js.pm

```diff
@@ -441,6 +446,94 @@ sub entity : Chained('root') PathPart('entity') Args(1)
     $c->res->body(encode_json($data));
 }
 
+Readonly our $MAX_FETCHED_ENTITIES => 25;
+
+Readonly our $MAX_FETCHED_ENTITIES_URL_LENGTH => (
+    # 36 => Length of an MBID.
+    # 24 => Number of separators.
+    ($MAX_FETCHED_ENTITIES * 36) + ($MAX_FETCHED_ENTITIES - 1)
+);
+
+sub entities : Chained('root') PathPart('entities') Args(2)
```

Comment:
This endpoint will be publicly accessible, even though it is neither promoted nor supported as public API. Is there anything to prevent potential abuse?

Reply:
Isn't that already the case with all of ws/js?

Reply:
Well, it's limited to 25 entities.  You can request far more information from /ws/2.  And yeah, all endpoints under /ws/js are public -- some are probably already used by userscripts.

Reply:
Yes it provides the same info as `/ws/js/entity` already does, but at a higher rate.
Rate limiting or statistics are not handled from this repository anyway, just asking.